hrms/payroll/doctype/payroll_entry/payroll_entry.js

```diff
@@ -136,9 +136,29 @@ frappe.ui.form.on('Payroll Entry', {
 			frm.add_custom_button(__("Create Salary Slips"), function() {
 				frm.trigger("create_salary_slips");
 			}).addClass("btn-primary");
+		} else if (
+			(!frm.doc.journal_entry_submitted
+			&& frm.doc.salary_slips_submitted)
+			|| (frm.doc.__onload && frm.doc.__onload.submitted_ss)
+		) {
+			frm.events.add_journal_entry_button(frm);
```

Comment:
You will have to remove accrual entry creation from `submit_salary_slips_for_employee`. It is currently created automatically on salary slip submission

Also remove accrual entry info from this message shown on submission since now its a separate action.

![image](https://github.com/frappe/hrms/assets/24353136/f05e18ad-1d12-4ed5-adec-9ab9e1982d92)